app/src/androidTest/java/com/github/sdpcoachme/database/CachingStoreTest.kt

```diff
@@ -585,36 +646,49 @@ class CachingDatabaseTest {
 
 
 
+
     private val exampleEmail = "example@email.com"
 
-    private val defaultUser = UserInfo(
-        "John",
-        "Doe",
-        exampleEmail,
-        "1234567890",
-        LAUSANNE,
-        false
-    )
+    companion object {
```

Comment:
To have all the test users in one place, you can just use the existing test users of UserInfoSamples. (change in next PR)